README.md

```diff
@@ -1,2 +1,183 @@
 # next-common-files
+
+[travis-url]:https://travis-ci.org/moxystudio/next-common-files
+[travis-image]:http://img.shields.io/travis/moxystudio/next-common-files/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/next-common-files
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-common-files/master.svg
+
 Next.js plugins that insert loaders for common files.
```

Comment:
> Next.js plugins that configures webpack with loaders for common files.

What do you think?

Reply:
If you change it, please update package.json's description and GitHub repo description

Reply:
Yes, that's better. The previous description didn't mention webpack, which, retrospectively, is a considerable lapse of context.